text/0000-default-field-values.md

```diff
@@ -0,0 +1,1658 @@
+- Feature Name: `default_field_values`
+- Start Date: 2018-12-17
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Allow `struct` definitions and `enum` variants to provide default values for
+individual fields and thereby allowing those to be omitted from initializers.
+When deriving `Default`, the provided values will then be used. For example:
+
+```rust
+#[derive(Default)]
+struct ExprType {
+    expr: Box<Expr>,
+    ty: Box<Ty>
+    attrs: Vec<Attribute> = Vec::new(),
+}
+```
+
+An attribute `#[default]` is also introduced which can be used on `enum`
+variants thereby allowing enums to work with `#[derive(Default)]`.
```

Comment:
`variants, thereby`

Yes, I'm a grammar Nazi. :-)

Reply:
Maybe worth mentioning here that this attribute feature is somewhat independent?